Adding the minimum AWS S3 policy needed for xbcloud to write/restore a backup.

mbenshoof opened the pull request:
This adds a sample AWS policy to the documentation for the least privilege to stream and restore a backup to S3 using xbcloud.

mbenshoof committed:
Document minimum AWS policy to backup with xbcloud

This is the minimum policy needed for xbcloud to write to S3.

mbenshoof committed:
Quick S3 typo fix

patrickbirch commented on docs/xbcloud/xbcloud_s3.md:
```diff
@@ -24,7 +24,39 @@ The following options are available when using Amazon S3:
-
+## Permissions setup
+
+Following the principle of "least-privilege", these are the minimum permissions needed for xbcloud to write backups to S3.
```
"these are the minimum permissions needed" - expected a list of permissions.

mbenshoof replied:
Fair enough - I was thinking that the policy would be enough, but I can summarize the permissions here on this line too.

mbenshoof committed:
Merge branch 'percona:8.0' into patch-1

mbenshoof committed:
Adding high level list of S3 permissions 

Offer a high level list of the permissions prior to posting the official IAM policy.

alina-derkach-oaza reviewed (approved):
LGTM
